docs/recent_activity/recent_activity.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Monitor recent activity logged actions.
+---
+
+# Recent activity log
+
+## Activity list
+
+On the Back Office, in Admin -> Activity list, you can consult what happens recently on the database.
+
+By default, the following actions are logged:
+
+- [Content](content_items.md) [create](create_edit_content_items.md#create-content-items), [publish](publish_instantly.md#publish-content-instantly), update, trash, recover, [delete](copy_move_hide_content.md#multi-file-move), delete translation, hide and reveal
+- Location create, delete, [hide](manage_locations_urls.md#hide-locations), reveal, update, move, [swap](manage_locations_urls.md#swap-locations) and subtree copy
+- Product [create](create_edit_product.md), update and delete
+- Product variant [create](work_with_product_variants.md#generate-variants), update and [delete](work_with_product_variants.md#delete-variants)
+- [Site](work_with_sites.md) [create](work_with_sites.md#create-a-website), [update](work_with_sites.md#edit-an-existing-website) and [delete](work_with_sites.md#delete-an-existing-website)
+
+By default, log entries are kept 30 days.
+
+Log entries are grouped by logical bonds like events, batches, or sessions.
+
+![Activity list](img/4.6_activity_list.png)
+
+### Filter activities
+
+On the right, a "Filters" block offers:
+
+* to only follow the activity of a user or a list of users,
+* to narrow the display to groups having given types of items, or actions.
+
+Select what you want to focus on and click the "Apply" button. Stop filtering by clicking the "Clear" button.
+
+The following setting example lists activity log groups containing a "publish" entry, and occurring withing a week from today.
+
+![Published last week](img/filters.png)
+
+## Dashboard block
+
+You can add a "Recent activity" block to your [dashboard](discover_ui.md#dashboard).
+
+!["Recent activity" block](img/recent_activity_block.png)
+
+You can set the block to display only log groups containing activities by some particular users, or concerning particular object classes.
+
+For example, the following Dashboard block focuses on Content and Location activity.
+
+!["Content recent activity" block settings](img/recent_activity_block_settings.png)
+
+## User profile
+
+On a user profile, recent activity of this user in shown.
```

Comment:
Remember that proper permissions are needed, so I guess we should also add a note here? The same applies to dashboard sentence above - without permissions, that dashboard block will not display iirc.

Reply:
Permission note and reminders added in 325bb266250a5771dbcebf9c61e74c89ebbfc189